app/views/static_pages/help.html.erb

```diff
@@ -1,16 +1,9 @@
 <% provide :title, t(".title") %>
-<!DOCTYPE html>
-<html>
-  <head>
-    <title><%= yield(:title) %> | <%= t ".railstutorialapp" %></title>
-  </head>
-  <body>
-    <h1><%= t ".title" %></h1>
-    <p>
-      <%= t ".gethelpon" %>
-      <%= link_to t(".railshelp"), Settings.helprails %>.
-      <%= t ".torailshelp" %>
-      <em><%= link_to t(".railstutorial"), Settings.bookrails %></em> <%= t ".book" %></a>.
-    </p>
-  </body>
-</html>
+<h1><%= t ".title" %></h1>
+<p>
+  <%= t ".gethelpon" %>
+  <%= link_to t(".railshelp"), Settings.helprails %>.
+  <%= t ".torailshelp" %>
+  <em><%= link_to t(".railstutorial"), Settings.bookrails %></em>
+  <%= t ".book" %></a>.
```

Comment:
Em, why you added this closing tag </a> ?

Reply:
Because I want to change "<a><em></em></a> to "link_to" tag. I forgot to delete it